spec/features/wikidata_course_spec.rb

```diff
@@ -23,7 +23,7 @@
 
   it 'shows Wikidata stats on the Home tab' do
     visit "/courses/#{course.slug}"
-    expect(page).to have_content('Wikidata stats')
+    expect(page).to have_content('www.wikidata.org')
```

Comment:
I'd like this PR to include a ruby feature spec for having multiple tabs and interacting with them. This file could be renamed `detailed course stats` or similar, and then the spec could be basically the same as this Wikidata one except with multiple tabs worth of data in the CoursesStats record.

The spec should test clicking through different tabs and seeing the different stats render.

Reply:
okay!